src/App.js

```diff
@@ -1,26 +1,35 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import {BrowserRouter as Router,
+        Switch, Route, Redirect} from 'react-router-dom';
+
+import PageTemplate from './components/pageTemplate';
+import Workouts from './pages/workouts';
+import Exercises from './pages/exercises';
+
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <>
+        <Switch>
+          <Redirect exact from='/' to='/workouts' />
+          <Route path="/workouts">
+            <PageTemplate title="Workouts">
+              <Workouts/>
+            </PageTemplate>
+          </Route>
+          <Route path="/exercises">
```

Comment:
/exercises to constants for all the project

Reply:
fixed